mbf_abstract_nav/src/abstract_planner_execution.cpp

```diff
@@ -354,7 +355,7 @@ namespace mbf_abstract_nav
           ROS_INFO_STREAM("New goal pose: (" << g.x << ", " << g.y << ", " << g.z << ")");
         }
 
-        make_plan = !(success || exceeded) || has_new_start_ || has_new_goal_;
```

Comment:
We should move this line into the while loop. Somehow it went down to the wrong place, I think.

Reply:
And then maybe remove the `planning_ = false` everywhere. That is much cleaner then. What do you say?

Reply:
These two flags are required. Otherwise, move_base action won't react to new goals